parse/PythonParser.cpp

```diff
@@ -98,13 +98,20 @@ PythonParser::PythonParser(PythonCommon& _python, const boost::filesystem::path&
             .def(py::self_ns::self * double())
             .def(py::self_ns::self * py::self_ns::self)
             .def(double() * py::self_ns::self)
+            .def(py::self_ns::self / py::self_ns::self)
             .def(py::self_ns::self + int())
             .def(py::self_ns::self + double())
             .def(py::self_ns::self + py::self_ns::self)
             .def(py::self_ns::self - py::self_ns::self)
+            .def(int() + py::self_ns::self)
```

Comment:
can you add something like `.def(int() + py::self_ns::self + int())` and then all permutations of operators and types?

Reply:
No, it doesn't work same way neither for Python nor for C++.